R/zchunk_L124.nonco2_unmgd_R_S_T_Y.R

```diff
@@ -43,80 +41,185 @@ module_emissions_L124.nonco2_unmgd_R_S_T_Y_DISABLED <- function(command, ...) {
-    GFED_Deforest_SO2 <- get_data(all_data, "emissions/GFED_Deforest_SO2")
-    GFED_ForestFire_CO <- get_data(all_data, "emissions/GFED_ForestFire_CO")
-    GFED_Deforest_CO <- get_data(all_data, "emissions/GFED_Deforest_CO")
-    GFED_ForestFire_NOx <- get_data(all_data, "emissions/GFED_ForestFire_NOx")
-    GFED_Deforest_NOx <- get_data(all_data, "emissions/GFED_Deforest_NOx")
+    EDGAR_sector <- get_data(all_data, "emissions/EDGAR/EDGAR_sector")
+    L124.LC_bm2_R_Grass_Yh_GLU_adj <- get_data(all_data, "temp-data-inject/L124.LC_bm2_R_Grass_Yh_GLU_adj")
+    L124.LC_bm2_R_UnMgdFor_Yh_GLU_adj <- get_data(all_data, "temp-data-inject/L124.LC_bm2_R_UnMgdFor_Yh_GLU_adj")
+    EDGAR_gases <- get_data(all_data, "EDGAR_gases")
+    GFED_ForestFire_SO2 <- get_data(all_data, "emissions/GFED/GFED_ForestFire_SO2")
+    GFED_Deforest_SO2 <- get_data(all_data, "emissions/GFED/GFED_Deforest_SO2")
+    GFED_ForestFire_CO <- get_data(all_data, "emissions/GFED/GFED_ForestFire_CO")
+    GFED_Deforest_CO <- get_data(all_data, "emissions/GFED/GFED_Deforest_CO")
+    GFED_ForestFire_NOx <- get_data(all_data, "emissions/GFED/GFED_ForestFire_NOx")
+    GFED_Deforest_NOx <- get_data(all_data, "emissions/GFED/GFED_Deforest_NOx")
 
-    # ===================================================
-    # TRANSLATED PROCESSING CODE GOES HERE...
-    #
-    # If you find a mistake/thing to update in the old code and
-    # fixing it will change the output data, causing the tests to fail,
-    # (i) open an issue on GitHub, (ii) consult with colleagues, and
-    # then (iii) code a fix:
-    #
-    # if(OLD_DATA_SYSTEM_BEHAVIOR) {
-    #   ... code that replicates old, incorrect behavior
-    # } else {
-    #   ... new code with a fix
-    # }
-    #
-    #
-    # NOTE: there are 'match' calls in this code. You probably want to use left_join_error_no_match
-    # For more information, see https://github.com/JGCRI/gcamdata/wiki/Name-That-Function
-    # NOTE: This code uses vecpaste
-    # This function can be removed; see https://github.com/JGCRI/gcamdata/wiki/Name-That-Function
-    # NOTE: This code uses repeat_and_add_vector
-    # This function can be removed; see https://github.com/JGCRI/gcamdata/wiki/Name-That-Function
-    # ===================================================
+    # Reformat temporary data -- Note: this can be removed after the upstream files are done
+    L124.LC_bm2_R_Grass_Yh_GLU_adj %>%
+      gather(year, value, -GCAM_region_ID, -Land_Type, -GLU) %>%
+      mutate(year = as.integer(substr(year, 2, 5))) ->
+      L124.LC_bm2_R_Grass_Yh_GLU_adj
+
+    L124.LC_bm2_R_UnMgdFor_Yh_GLU_adj %>%
+      gather(year, value, -GCAM_region_ID, -Land_Type, -GLU) %>%
+      mutate(year = as.integer(substr(year, 2, 5))) ->
+      L124.LC_bm2_R_UnMgdFor_Yh_GLU_adj
+
+    # Prepare EDGAR emissions for use
+    # Map in region ID and sector name; extrapolate to 2010; aggregate to sector & region; convert from Gg to Tg
+    EDGAR_gases %>%
```

Comment:
@kvcalvin - this is also quite an expensive step. Is it possible to defer the ISO code stuff until after line 98? It doesn't look like it's needed until after the `filter` in that line, which drops the size of `EDGAR_gases` by almost 90%.